pkg/scheduler/watchers/clusterschedulingpolicysnapshot/controller_integration_test.go

```diff
@@ -248,3 +260,168 @@ var _ = Describe("cluster scheduling policy snapshot scheduler source controller
 		})
 	})
 })
+
+var _ = Describe("scheduling policy snapshot scheduler source controller for ResourcePlacement", Serial, Ordered, func() {
```

Comment:
Add a comment about why need Serial here? It's because the keycollector is shared right?